src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -783,6 +729,72 @@ PMComplexTest >> testTanh [
 	self assert: (c2 imaginary closeTo: c tanh imaginary).
 ]
 
+{ #category : #'testing - complex conjugation' }
+PMComplexTest >> testThatComplexConjugateHasEqualRealPartAndAnImaginaryPartWithEqualMagnitudeButOppositeSign [
+	| z |
+	z := 3 + 2 i.
+	
+	self assert: z complexConjugate equals: 3 - 2 i
```

Comment:
I think that the code would be more readable if we don't create a variable here:
```st
self assert: (3 + 2i) complexConjugate equals: (3 - 2i).
```
Complex numbers are native to Pharo, they should be used like any other numbers.
And personally, I wouldn't write this:
```st
| a b |
a := 4.
b := -4.

self assert: a negated equals: b.
```
I would just write
```st
self assert: 4 negated equals: -4
```
So the same logic should apply to complex numbers

Reply:
To sum up, I propose the following:
```st
testComplexConjugate

    self assert: (3 + 2i) complexConjugate equals: (3 - 2i).
```

Reply:
The method with that name already exists but it is not good...
```st
testComplexConjugate
	| z complexConjugateOfZ expected |
	z := 5 - 6 i.
	
	complexConjugateOfZ := z complexConjugate .
	
	expected := 5 + 6 i.
	self assert: complexConjugateOfZ equals: expected.
```
I propose to replace it with a more readable one-line version:
```st
testComplexConjugate

	self assert: (5 - 6i) complexConjugate equals: (5 + 6i).
```

Reply:
I will open a separate issue for that

Reply:
Issue #246